### ISSUE TITLE
Use svySearch in combination with foundset filters

### ISSUE THREAD

**Issue by saleman67** [Servoy/svySearch#18]
I am looking trough the options but its not clear to me how the use the svySearch in combination with foundset filters. If I have foundset filters on my foundset they seem to get lost.

**Comment by og1** [comment]
It works for me. I have a contact table with a foundset filter on a boolean archive field. My search fields are: contact_no, name,  contact_to_address.city, contact_to_country.country. The foundset filter is respected by svySearch and included in the query as expected.

**Comment by seanthomasdevlin** [comment]
Hi Sanneke. The module absolutely can work with foundset filters.
Please see SimpleSearch.[loadRecords](https://github.com/Servoy/svySearch/wiki#loadrecords)(foundset)
If the foundset already has filters applied, they will be respected. Under the hood, it is really just foundset.loadRecords(QBSelect).

**Comment by saleman67** [comment]
Hi Sean,
The problem is that a foundset.loadRecords(QBSelect) does not respect foundset filters (not table filters).
This way you cant use search in combination with foundset filters.

Regards Sanneke

this.loadRecords = function(foundSet){
		return foundSet.loadRecords(this.getQuery());
	}

**Comment by seanthomasdevlin** [comment]
Hi Sanneke,
I cannot reproduce the issue you report. If a foundset has filters applied, then subsequent calls to loadRecords will respect the filter. Do you first apply the filter to the foundset? 

Please re-open and send a code sample if you still have this issue

**Comment by saleman67** [comment]
Hi Sean,
By making a sample I found what the problem is. Because you where so sure it should work correct I thought there must be something special, and there is. It is because I put a named foundset on the form, without the name it works fine, with the name, foundset filters are gone. I made a case in Jira, I geuss its a namedFoundset bug. If you want to see the sample: https://support.servoy.com/browse/SVY-12840
